docs/fundamentals/systems/form/validate.md

```diff
@@ -130,6 +130,35 @@ class EqualsText extends Validator {
 }
 ```
 
+The `execute()` method returns a boolean by default, but it can also be customized to return enums instead. In this example below, we customize the method to return one among some possible enums as strings.
+
+```js
+class EnumOutComeValidator extends Validator {
+  static validatorName = 'EnumOutCome';
+
+  execute(modelValue, maxLength) {
+    if (modelValue.length > maxLength) {
+      return 'overload';
+    } else if (modelValue.length < 0) {
+      return 'error';
+    }
+    return 'ok';
```

Comment:
In case there's no error, it should return false: https://github.com/ing-bank/lion/blob/master/packages/input-tel/src/validators.js#L34 
(not really clear from this code, so that's why it's good we have an explicit demo ;))